test/lib/extractor/image.test.ts

```diff
@@ -1,3 +1,7 @@
+#!/usr/bin/env node_modules/.bin/ts-node
+// Shebang is required, and file *has* to be executable: chmod +x file.test.js
+// See: https://github.com/tapjs/node-tap/issues/313#issuecomment-250067741
+
```

Comment:
🤦‍♂ we would want to address this silliness at some point... maybe move to Jest while at it?
awesome find 👍

Reply:
I dream about moving to Jest every night